test/e2e/kwokctl/dryrun/testdata/binary/create_cluster.txt

```diff
@@ -6,16 +6,16 @@ mkdir -p <ROOT_DIR>/workdir/clusters/<CLUSTER_NAME>/pki
 # Generate PKI to <ROOT_DIR>/workdir/clusters/<CLUSTER_NAME>/pki
 mkdir -p <ROOT_DIR>/workdir/clusters/<CLUSTER_NAME>/etcd
 # Download https://github.com/etcd-io/etcd/releases/download/v3.5.15/etcd-v3.5.15-<OS>-<ARCH>.<TAR> and extract etcd to <ROOT_DIR>/workdir/clusters/<CLUSTER_NAME>/bin/etcd
-# Download https://dl.k8s.io/release/v1.31.0/bin/<OS>/<ARCH>/kube-apiserver to <ROOT_DIR>/workdir/clusters/<CLUSTER_NAME>/bin/kube-apiserver
-# Download https://dl.k8s.io/release/v1.31.0/bin/<OS>/<ARCH>/kube-controller-manager to <ROOT_DIR>/workdir/clusters/<CLUSTER_NAME>/bin/kube-controller-manager
-# Download https://dl.k8s.io/release/v1.31.0/bin/<OS>/<ARCH>/kube-scheduler to <ROOT_DIR>/workdir/clusters/<CLUSTER_NAME>/bin/kube-scheduler
+# Download https://github.com/kwok-ci/k8s/releases/download/v1.31.0-kwok.0-<OS>-<ARCH>/kube-apiserver to <ROOT_DIR>/workdir/clusters/<CLUSTER_NAME>/bin/kube-apiserver
+# Download https://github.com/kwok-ci/k8s/releases/download/v1.31.0-kwok.0-<OS>-<ARCH>/kube-controller-manager to <ROOT_DIR>/workdir/clusters/<CLUSTER_NAME>/bin/kube-controller-manager
+# Download https://github.com/kwok-ci/k8s/releases/download/v1.31.0-kwok.0-<OS>-<ARCH>/kube-scheduler to <ROOT_DIR>/workdir/clusters/<CLUSTER_NAME>/bin/kube-scheduler
```

Comment:
Revert this

Reply:
Why did this happen, even on the clean `main` branch?

Reply:
https://kwok.sigs.k8s.io/docs/user/kwokctl-platform-specific-binaries/

k8s doesn't officially provide Darwin binaries, the binary runtime under Darwin uses unofficial builds on https://github.com/kwok-ci/k8s by default.